src/TableActionCell/TableActionCell.js

```diff
@@ -55,14 +56,22 @@ function renderHiddenActions(actions, popoverMenuProps) {
     <PopoverMenu
```

Comment:
The `<PopoverMenu/>` was always opened to the top before. Personally I prefer it automatically, but I would stick with the previous behaviour:
```suggestion
     <PopoverMenu
      placement="top"
```